io-template.yml

```diff
@@ -109,15 +110,16 @@ IO_WORKFLOWENGINE:
         --jira.assignee=$JIRA_ASSIGNEE 
         --jira.enable=$ENABLE_JIRA 
         --rally.api.url=$RALLY_URL 
-        --rally.project.name=$RALLY_PROJECT_KEY
+        --rally.project.name=$RALLY_PROJECT_KEY 
```

Comment:
Can we remove extra space here and other places as well?

Reply:
Extra white space is intentionally added, so that if someone removes indentation somehow, we shall need to keep space between the passed options.